src/focus-change.cpp

```diff
@@ -124,11 +121,11 @@ class wayfire_focus_change_t : public wf::plugin_interface_t
             const int32_t bias_right = grace_right.value();
             const int32_t bias_left  = grace_left.value();
 
-            const bool w_cond = cx >= scan_w_l && cx <= scan_w_h;
-            const bool h_cond = cy >= scan_h_l && cy <= scan_h_h;
+            const bool w_cond = cx + cxr >= scan_w_l && cx - cxr <= scan_w_h;
+            const bool h_cond = cy + cyr >= scan_h_l && cy - cyr <= scan_h_h;
 
             bool contains    = false;
-            int32_t distance = 0;
+            int32_t distance = INT32_MAX;
```

Comment:
What is the reasoning behind switching to INT32_MAX and using it in other places? Would initializing to 0 make things not work correctly?

Reply:
there was no difference per say, it was just to make it consistent with choices I made elsewhere.